catalogue/webapp/pages/workv2.js

```diff
@@ -17,14 +17,28 @@ import Button from '@weco/common/views/components/Buttons/Button/Button';
 import MetaUnit from '@weco/common/views/components/MetaUnit/MetaUnit';
 import {workLd} from '@weco/common/utils/json-ld';
 import WorkMedia from '../components/WorkMedia/WorkMedia';
+import WorkDrawer from '@weco/common/views/components/WorkDrawer/WorkDrawer';
 import {getWork} from '../services/catalogue/worksv2';
 import {worksV2Link} from '../services/catalogue/links';
+import getLicenseInfo from '@weco/common/utils/get-license-info';
 
 export type Link = {|
   text: string;
   url: string;
 |};
 
+function constructLicenseString(licenseType) {
+  const licenseInfo = getLicenseInfo(licenseType);
+  return `<a href="${licenseInfo.url}">${licenseInfo.text}</a>`;
+}
+
+function constructAttribution(work, contributors, credit, licenseType, canonicalUri) {
+  const title = work.title ? `'${work.title}' ` : '';
+  // const contributorsString = '';
+  const license = constructLicenseString(licenseType);
+  return [`${title}. Credit: <a href="${canonicalUri}">${credit}</a>. ${license}`];
```

Comment:
Is there a reason not to convert this to JSX?

Reply:
nope